<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartManager</class>
 <widget class="QWidget" name="CartManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Корзина</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="cartlabel">
     <property name="text">
      <string>Купленные билеты</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="cartList"/>
   </item>
   <item>
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="doneButton">
     <property name="text">
      <string>Оформить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
